internal/bits/count_amd64.s

```diff
@@ -31,7 +31,7 @@ TEXT ·countByte(SB), NOSPLIT, $0-40
     SHRQ $8, DX
     SHLQ $8, DX
     ADDQ AX, DX
-    VPBROADCASTB value+24(FP), Z0
+    VPBROADCASTB BX, Z0
```

Comment:
I don't see where we are setting BX but just going to assume one of the instructions manipulates it and you know what you're doing here haha.